contrib/ansible/deploy-devel-playbook.yml

```diff
@@ -66,6 +74,8 @@
         CLUSTER_API_IMAGE_PULL_POLICY: "{{ cluster_api_image_pull_policy }}"
         MACHINE_CONTROLLER_IMAGE: "{{ machine_controller_image }}"
         MACHINE_CONTROLLER_IMAGE_PULL_POLICY: "{{ machine_controller_image_pull_policy }}"
+        ANSIBLE_IMAGE: "{{ ansible_image }}"
+        ANSIBLE_IMAGE_PULL_POLICY: "{{ ansible_image_pull_policy }}"
```

Comment:
It might be a good idea to use debug output to print a big WARNING on what type of deployment was used and what it means, to help folks catch if they did something unintentional.